packages/@aws-cdk-testing/framework-integ/test/aws-ses-actions/test/integ.actions.ts

```diff
@@ -13,14 +13,21 @@ import { STANDARD_NODEJS_RUNTIME } from '../../config';
  * 1. Create a free Workmail test domain (https://us-east-1.console.aws.amazon.com/workmail/v2/home?region=us-east-1#/organizations/create)
  *  - It should automatically be added to your list of verified SES domains, no need to exit the SES sandbox
  * 2. Add a new user email address in the Workmail console
- * 3. Update the TEST_EMAIL constant with the email address of the user you created
- * 4. Deploy the stack with --no-clean, and send an email to the email address you created
- * 5. Check the following:
+ * 3. Remove the automatically created INBOUND_MAIL rule set (https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/email-receiving/INBOUND_MAIL)
+ * 4. Update the TEST_* constants declared below with the organization details and email address you created
+ * 5. Deploy the stack with --no-clean
+ * 6. Set the rule set as active in the SES web console (https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/email-receiving)
+ * 7. Send an email to the email address you created
+ * 8. Check the following:
+ *  - The email should be in the WorkMail user inbox
  *  - The email should be saved to the S3 bucket
- *  - The SQS queue should receive receipt notifications
+ *  - The SQS queue should receive 6 receipt notifications
+ *  - A bounce notification should be sent back to the sender mailbox
  */
 
-const TEST_EMAIL = 'test@cdk-test-123.awsapps.com';
+const TEST_WORKMAIL_DOMAIN = 'cdk-test-123.awsapps.com';
+const TEST_ORGANIZATION_ARN = 'arn:aws:workmail:us-east-1:339712719728:organization/m-5ea60ed9e37442c388898996f05c17ac';
```

Comment:
Curious whose account is `339712719728`?

Reply:
Mine 😅 If a maintainer wants to update the snapshot with a CDK team account feel free to update them. I assumed it would be easier for everyone to let whoever updates this particular snapshot would update these constants

Reply:
I'm fine leaving it here since we will always test the snapshot if there's no change. But please make sure to remove the bucket yourself to avoid any potential issue.